packages/inference/src/providers/replicate.ts

```diff
@@ -62,7 +62,21 @@ abstract class ReplicateTask extends TaskProviderHelper {
 	}
 }
 
+
+
 export class ReplicateTextToImageTask extends ReplicateTask implements TextToImageTaskHelper {
+	override preparePayload(params: BodyParams): Record<string, unknown> {
+		const payload = super.preparePayload(params);
+
+		// For Flux LoRAs, use black-forest-labs/flux-dev-lora
+		if (params.mapping?.adapter === "lora" && params.mapping.adapterWeightsPath) {
+			payload.input.model = "black-forest-labs/flux-dev-lora";
+			payload.input.lora_weights = `${HF_HUB_URL}/${params.mapping.hfModelId}`;
+		}
```

Comment:
from what I see in your documentation: https://replicate.com/black-forest-labs/flux-dev-lora/api/schema (Flux dev loras) and https://replicate.com/fermatresearch/sdxl-controlnet-lora/api/schema (SDXL loras), we don't need to send the model name in the payload, no? 
```suggestion
		if (params.mapping?.adapter === "lora" && params.mapping.adapterWeightsPath) {
			payload.input.lora_weights = `${HF_HUB_URL}/${params.mapping.hfModelId}`;
		}
```
Small note: if you want to send a specific payload for Flux LoRAs, you need to add a condition : `if (params.mapping.providerId === "black-forest-labs/flux-dev-lora") {...}`